### PR TITLE
More verbose status

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/12612
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by SabaPing — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: tiup/tiup-command-status.md [hunk 1]
```diff
@@ -36,4 +36,20 @@ A table consisting of the following fields:
 - `Binary`: The binary file path of the components.
 - `Args`: The starting arguments of the operating components.
 
+### Component status
+
+A component can run in one of the following statuses:
+
+- Up: The component is running properly.
```
Comment (dveeden) [inline tiup/tiup-command-status.md:43]:
```suggestion
- Up: The component is running normaly.
```

File: tiup/tiup-command-status.md [hunk 2]
```diff
@@ -36,4 +36,20 @@ A table consisting of the following fields:
...
+- Down or Unreachable: The component is not started or a network problem exists on the corresponding host.
```
Comment (dveeden) [inline tiup/tiup-command-status.md:44]:
```suggestion
- Down or Unreachable: The component is not running or a network problem exists on the corresponding host.
```

File: tiup/tiup-component-cluster-display.md [hunk 3]
```diff
@@ -64,4 +64,20 @@ tiup cluster display <cluster-name> [flags]
     - Data Dir: the data directory of the service. `-` means no data directory.
     - Deploy Dir: the deployment directory of the service
 
+### Node service status
+
+A node service can run in one of the following statuses:
+
+- Up: The node service is running properly.
+- Down or Unreachable: The node service is not started or a network problem exists on the corresponding host.
```
Comment (dveeden) [inline tiup/tiup-component-cluster-display.md:72]:
```suggestion
- Down or Unreachable: The node service is not running or a network problem exists on the corresponding host.
```

File: tiup/tiup-component-cluster-display.md [hunk 4]
```diff
@@ -64,4 +64,20 @@ tiup cluster display <cluster-name> [flags]
     - Data Dir: the data directory of the service. `-` means no data directory.
     - Deploy Dir: the deployment directory of the service
 
+### Node service status
+
+A node service can run in one of the following statuses:
+
+- Up: The node service is running properly.
```
Comment (dveeden) [inline tiup/tiup-component-cluster-display.md:71]:
```suggestion
- Up: The node service is running normaly.
```

File: dashboard/dashboard-cluster-info.md [hunk 5]
```diff
@@ -35,7 +35,9 @@ The list includes the following information:
 - Deployment directory: The directory in which the instance binary file is located.
 - Git Hash: The Git Hash value corresponding to the instance binary file.
 
-An instance has the following running status:
+### Instance status
+
+An instance can run in one of the following statuses:
 
 - Up: The instance is running properly.
```
Comment (shichun-0415) [inline dashboard/dashboard-cluster-info.md:42]:
```suggestion
- Up: The instance is running normaly.
```

File: dashboard/dashboard-cluster-info.md [hunk 6]
```diff
@@ -35,7 +35,9 @@ The list includes the following information:
...
 - Down or Unreachable: The instance is not started or a network problem exists on the corresponding host.
```
Comment (shichun-0415) [inline dashboard/dashboard-cluster-info.md:43]:
```suggestion
- Down or Unreachable: The instance is not running or a network problem exists on the corresponding host.
```

**Pull request merged**